utils/contract-utils.js

```diff
@@ -14,6 +16,32 @@ const isValidContract = function (contract) {
 	return true;
 };
 
+const isValidBytecode = function (libraries, bytecode) {
```

Comment:
Is valid bytecode should not need the libraries field. It should only possibly check if there is any occurrences of __ and return false if there are some